src/components/tiles/geometry/geometry-tile.scss

```
@@ -19,6 +19,7 @@ $toolbar-width: 44px;
     }
     .geometry-size-me {
       height: 100%;
+      overflow: clip;
```

Based on our story hour conversation on the topic, may want to make this:
```suggestion
      overflow: hidden;
      overflow: clip;
```
to handle browsers that don't support `clip` yet.